pkg/core/hostid/aws.go

```diff
@@ -44,7 +44,7 @@ func AWSUniqueID(cloudMetadataTimeout timeutil.Duration) string {
 	if err != nil {
 		log.WithFields(log.Fields{
 			"error": err,
-		}).Debug("Failed to unmarshal AWS instance-identity response")
+		}).Debugf("Failed to unmarshal AWS instance-identity response. Body: %s", body)
```

Comment:
I would add a field "body" that has it instead of adding it to the message itself.

Reply:
done